packages/select/src/components/select/dropdownRenderer.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as React from "react";
+
+/**
+ * An object describing how to render the contents of a dropdown.
+ * A `dropdownRenderer` receives this object as its sole argument.
+ */
+export interface IDropdownRendererProps<T> {
```

Comment:
per your extra thought in description, i think `MenuRenderer` is the name we're looking for, and the default implementation will simply render a `Menu` containing the items. this is also consistent with how we [talk about dropdowns](http://blueprintjs.com/docs/v2/#core/components/menu.dropdown-menus) elsewhere in Blueprint: as a composition of `Popover` and `Menu`.